docs/source/release/v6.1.0/framework.rst

```diff
@@ -5,6 +5,10 @@ Framework Changes
 .. contents:: Table of Contents
    :local:
 
+.. figure:: ../../images/Multiple_Scatter_Intensity.png
```

Comment:
Could you make it clear what entry this image is connected to. Perhaps by making the entry bold.